e2e/cypress/integration/channel/message_permalink_spec.js

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
non-blocking: I forgot to comment to move this to ` e2e/cypress/integration/messaging`.